mon-pix/tests/integration/components/navbar-header-test.js

```diff
@@ -13,7 +13,7 @@ describe('Integration | Component | navbar-header', function() {
   context('when user is on desktop', function() {
     beforeEach(async function() {
       setBreakpoint('desktop');
-      await render(hbs`{{navbar-header media=media}}`);
+      await render(hbs`<NavbarHeader></NavbarHeader>`);
```

Comment:
En one line, j'ai tendance à préférer qui me parait plus lisible
```suggestion
      await render(hbs`<NavbarHeader />`);
```